sos/report/plugins/kafka.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2024 Alejandro Santoyo <alejandro.santoyo@canonical.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import os
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Kafka(Plugin, UbuntuPlugin):
+    """
+    No current deb for Kafka but a tarball to download, decompress and
+    use. Two available snaps, i.e., `kafka` and `charmed-kafka`, with
+    the former not actively maintained anymore, so focusing on the
+    latter.
+    """
+
+    short_desc = 'Apache Kafka plugin'
+    plugin_name = 'kafka'
+    profiles = ('services',)
+    services = ('',)
```

Comment:
I'm not sure what this is intended to do, but if there are no services to check for enablement, just remove this line rather than passing a tuple with an empty string.

Reply:
Ah yes, no need for that, forgot to remove it.